package-system/OpenEXR/FindImath.cmake

```diff
@@ -14,13 +14,46 @@ include(${CMAKE_CURRENT_LIST_DIR}/o3de_package_utilities.cmake)
 
 o3de_import_existing_config_files(Imath ${CMAKE_CURRENT_LIST_DIR}/OpenEXR/lib/cmake)
 
-o3de_import_targets(NAMESPACE_FROM 
-                        Imath
-                    NAMESPACE_TO 
-                        3rdParty
-                    COMPONENTS 
-                        Imath 
-                        ImathConfig)
+set(Imath_COMPONENTS
+    Imath::Imath
+    Imath::ImathConfig
+)
+
+foreach(component ${Imath_COMPONENTS})
+    if(TARGET ${component})
+        # convert the includes to system includes
+        get_target_property(system_includes ${component} INTERFACE_INCLUDE_DIRECTORIES)
+        set_target_properties(${component} PROPERTIES INTERFACE_INCLUDE_DIRECTORIES "") # Clear it in case someone refers to it
+        
+        if (COMMAND ly_target_include_system_directories)
+            ly_target_include_system_directories(TARGET ${component} INTERFACE ${system_includes})
+        else()
+            target_include_directories(${component} SYSTEM INTERFACE ${system_includes})
+        endif()
+
+        # Alias the target with 3rdParty prefix
+        add_library(3rdParty::${component} ALIAS ${component})
+
+        # inside the loop where it sets the system includes for each component
+        foreach(conf IN LISTS CMAKE_CONFIGURATION_TYPES)
+            string(TOUPPER ${conf} UCONF)
+            if (${UCONF} STREQUAL "DEBUG" AND ${CMAKE_SYSTEM_NAME} STREQUAL Windows)
+                set_target_properties(${component} PROPERTIES 
+                                        MAP_IMPORTED_CONFIG_${UCONF} DEBUG)
+            else()
+                set_target_properties(${component} PROPERTIES 
+                                        MAP_IMPORTED_CONFIG_${UCONF} RELEASE)
+            endif()
+        endforeach()
+    else()
+        message(WARNING "Target not found in Imath: ${component}")
+    endif()
+endforeach()
+
+# create main library alias that O3DE can use to get default dependencies.
+# users can also bind to any of the other components above, but this one gives you most of the functionality
+# you probably want:
+add_library(3rdParty::Imath ALIAS Imath::Imath)
```

Comment:
Somethings not right here, this is a reversion to what it used to be?

Reply:
Good catch, reverted!